flutter_vlc_player/example/pubspec.yaml

```diff
@@ -1,19 +1,40 @@
-name: flutter_vlc_player_example
-description: Demonstrates how to use the flutter_vlc_player plugin.
-publish_to: 'none'
+name: fluttervlcplayerexample
+description: A new Flutter project.
```

Comment:
It doesn't look right - should be reverted?

Reply:
You are right. Let me check that.

Reply:
Reason for the update was the error: `The lower bound of "sdk: '>=2.3.0 <3.0.0'" must be 2.12.0 or higher to enable null safety.
`
Creating a new example by `flutter create --org software.solid flutter_vlc_player_example` was the best option.
The new example (optional but recommend) needs also  few updates e.g. the build tools or gradle version.

Reply:
Hmm. You dont need to regenerate a project just to update the SDK bounds. This project already supported null safety per https://github.com/solid-software/flutter_vlc_player/pull/205/files 


Im all for updated examples, but killing and regenerating the example project is completely unnecessary and this change should be reverted.  You can make all the gradle / xcode changes without regen.

Reply:
The link you mention just updated the plugin and not the example.

Reply:
Ah I am following now... Ok this makes sense.